LandSandBoat/server
# [lua] Refactor Additional Effects to resolve cyclomatic complexity

**_I affirm:_**

## What does this pull request do?

Moves additional effects into global functions that can be called from elsewhere
Remove cyclomatic complexity call
Also fix Onyos knife

## Steps to test these changes

Test every single add effect type and see no error
This PR does NOT fix broken things that have no visuals!
